### PR TITLE
Adding Incident Tests & update versions

### PR DESCRIPTION
**Pull request by yonetmen**

### CODE REVIEW

**Comment by yonetmen** [comment]
test me

File: pom.xml [hunk 1]
```diff
@@ -49,6 +49,7 @@
 		<unionvms.project.plugins.flux-rest>1.1</unionvms.project.plugins.flux-rest>
 		<unionvms.project.stream-collector>0.9.12</unionvms.project.stream-collector>
 		<unionvms.project.hawtio-wildfly>2.7.1</unionvms.project.hawtio-wildfly>
+		<uvms.commons.version>4.0.13</uvms.commons.version>
```
Comment (peerik) [inline pom.xml]: Where is this import used?

**Review by peerik — APPROVED** [review]

**Comment by yonetmen** [comment]
test me

**Comment by yonetmen** [comment]
test me

**Pull request merged**